components/collective-page/sections/Location.js

```diff
@@ -48,9 +48,18 @@ const Location = ({ collective: event, refetch }) => {
             <P fontWeight="bold" fontSize="18px">
               <FormattedMessage id="event.privateInstructions.label" defaultMessage="Private instructions" />
             </P>
-            <P mt={3} fontSize="14px">
+            <pre
+              style={{
+                whiteSpace: 'pre-wrap',
+                fontSize: '14px',
+                marginTop: 3,
+                background: 'transparent',
+                border: 'none',
+                fontFamily: "'Inter', sans-serif",
+              }}
+            >
```

Comment:
Styling must be done through styled-components. Simpler alternative:

```suggestion
            <P mt={3} fontSize="14px"  whiteSpace="pre-wrap">
```

Reply:
This works well. Also updated the screenshot in the description with this change.